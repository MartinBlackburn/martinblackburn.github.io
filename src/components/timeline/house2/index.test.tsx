// libraries
import { render } from "@testing-library/react";

// component under test
import House1Timeline from ".";

describe("House1Timeline component", () => {
    test("without dark mode", () => {
        const { asFragment } = render(<House1Timeline />);

        expect(asFragment()).toMatchSnapshot();
    });
});
