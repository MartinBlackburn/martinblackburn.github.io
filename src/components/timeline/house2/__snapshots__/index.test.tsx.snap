// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`House1Timeline component without dark mode 1`] = `
<DocumentFragment>
  <div
    class="timeline timeline--dark"
  >
    <h1
      class="timeline__title"
    >
      House 2
    </h1>
    <div
      class="timeline__content"
    >
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Offer accepted
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-03-15"
          >
            15 March 2022
          </time>
            
        </div>
        <p>
          We got pretty lucky and a house came up not too long after we lost the last one. After a little bidding war we won! Although it was £40,000 over asking this time, eeek.
        </p>
        <p>
          They already have an house offer accepted, a new build, but we do have to wait until august for that to be completed.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Mortgage down valued
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-05-09"
          >
            09 May 2022
          </time>
            
        </div>
        <p>
          The mortgage was rejected, they didn't think the house was worth our offer.
        </p>
        <p>
          Our mortgage broker said this has happened a lot recently with this particular mortgage provider and we should try another.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Mortgage down valued again
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-05-19"
          >
            19 May 2022
          </time>
            
        </div>
        <p>
          The mortgage was rejected with another lender. Uh oh.
        </p>
        <p>
          We now have two options, make up the difference ourselves (£40,000), or negotiate the price.
        </p>
        <p>
          The seller has refused to reduce the price, so we agreed to make up the difference so long as the survey came back ok.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Survey results
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-06-13"
          >
            13 June 2022
          </time>
            
        </div>
        <p>
          Not good news, the survey listed several major issues, roof replacement, conservatory replacement, penetrating damp on the first floor, rising damp on the ground floor, a crack in the external wall, plus a few other smaller issues.
        </p>
        <p>
          So we have sent the survey to the seller and have said that we are reducing our offer based on the work needed.
        </p>
        <p>
          After several quotes, the work looks like it could be anywhere from £20,000 to £60,000 to do.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Price reduction
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-06-29"
          >
            29 June 2022
          </time>
            
        </div>
        <p>
          With a lot of back and forth, we ended up with a £25,500 price reduction. This also reduces our stamp duty bill by £1,275. So a total of £26,775 saved that we can use towards repairs. Hopefully enough to cover most of the work needed.
        </p>
        <p>
          It does mean we are still paying £15,000 over the value of the house, but as it's a house for life, we feel that it's worth it.
        </p>
        <p>
          We may not be able to do all the repairs/decorating we wanted in one go due to the cost. However, the discount will allow us to do the major ones, such as the roof replacement and fix the damp problems.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Initial report
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-07-04"
          >
            04 July 2022
          </time>
            
        </div>
        <p>
          We have the initial report from our solicitor. It's outstanding a few enquires from the sellers solicitor, but we should get these soon.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Still outstanding enquires
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-07-21"
          >
            21 July 2022
          </time>
            
        </div>
        <p>
          Still waiting on enquiries from the solicitors. Not even sure what they think they are missing at this point, there is very little communication from either solicitor.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Ready to sign?
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-08-01"
          >
            01 August 2022
          </time>
            
        </div>
        <p>
          Our solicitor just said we'll have the contracts ready to sign buy the end of the week.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Nup not ready
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-08-11"
          >
            11 August 2022
          </time>
            
        </div>
        <p>
          Despite what we were told last week, our solicitor clearly never looked at the file and we are still missing some things from the sellers solicitor.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Talk to each other
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-08-12"
          >
            12 August 2022
          </time>
            
        </div>
        <p>
          The estate agent doesn't think there was anything out standing, but our solicitor does.
        </p>
        <p>
          So we have copied them both in on an email asking what they are missing and who needs to provide it.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Different name
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-08-17"
          >
            17 August 2022
          </time>
            
        </div>
        <p>
          Turns out the seller was married after buying the house. So the title deeds with land registry don't reflect her current name. I thought this would have been a basic thing they should have checked at the start, but hey!
        </p>
        <p>
          So the seller will need to update land registry before we can carry on with the purchase.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Autograph time?
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-09-06"
          >
            06 September 2022
          </time>
            
        </div>
        <p>
          We have finally got our contracts and can now sign and send them off to the solicitors.
        </p>
        <p>
          Actually no, it's full of mistakes and so need to have them amended. Obvious mistakes too, like: the wrong purchase price, wrong stamp duty amount, wrong address for my contact details to name a few.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Now it's autograph time
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-09-07"
          >
            07 September 2022
          </time>
            
        </div>
        <p>
          We have finally got our 
          <em>
            correct
          </em>
           contracts and can now sign and send them off to the solicitors.
        </p>
        <p>
          No completion date yet, but we finally feel like we might actually get to move in some time this year.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Can we move yet?
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-10-15"
          >
            15 October 2022
          </time>
            
        </div>
        <p>
          There has been lots of back and forth over the last month, we don't seem to be any closer to exchanging either.
        </p>
      </div>
      <div
        class="timelineItem timelineItem--dark"
      >
        <h2
          class="timelineItem__title"
        >
          Exchange time
        </h2>
        <div
          class="timelineItem__date"
        >
          <time
            datetime="2022-11-17"
          >
            17 November 2022
          </time>
            
        </div>
        <p>
          We have finally exchanged and have a completion date.
        </p>
        <p>
          We'll be getting our keys on next week. Wooooooo!!!
        </p>
        <p>
          Now the fun starts of getting repairs and decorating, as well as organising all the boring, but essential things, like insurance.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
